src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -201,8 +216,8 @@ public boolean check(final ItemType t) {
 	}
 	
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
-		return "dispense/spawn/drop/craft/pickup/consume/break/despawn/merge" + (types == null ? "" : " of " + types);
+	public String toString(@Nullable Event event, boolean debug) {
+		return "dispense/spawn/drop/craft/pickup/consume/break/despawn/merge/move item" + (types == null ? "" : " of " + types);
```

Comment:
Not sure if `item` part is required because it's already for item events and has `of type`
```suggestion
		return "dispense/spawn/drop/craft/pickup/consume/break/despawn/merge/move" + (types == null ? "" : " of " + types);
```